### PR TITLE
circle ci and package updates

### PR DESCRIPTION
**Pull request by fivetran-jamie**
- add circle ci testing
- add custom schemas
- update readme
- postgres compatibility re https://github.com/fivetran/dbt_mixpanel/issues/8

### CODE REVIEW

**Review by fivetran-joemarkiewicz — COMMENTED** [review]
Since we are initiating CircleCi for the first time in this package do you think it would be beneficial to incorporate Postgres compatibility? 

I looked through the models and it seems you may need to make some changes within the date spine and possibly window functions.

File: README.md [hunk 1]
```diff
@@ -192,17 +192,35 @@ vars:
...
+```
+
 ## Contributions
 Additional contributions to this package are very welcome! Please create issues
 or open PRs against `master`. Check out 
 [this post](https://discourse.getdbt.com/t/contributing-to-a-dbt-package/657) 
 on the best workflow for contributing to a package.
 
+## Database Support
+This package has been tested on BigQuery, Snowflake and Redshift.
```
Comment (fivetran-joemarkiewicz) [inline README.md:216]: If this ends up working for the customer we should update Postgres to be included.

File: macros/pivot_event_properties_json.sql [hunk 2]
```diff
@@ -9,6 +9,9 @@ replace(parse_json(event_properties):"{{ property }}", '"', '')
 
 {%- elif target.type == 'redshift' -%}
 replace(json_extract_path_text(event_properties, {{ "'" ~ property ~ "'" }} ), '"', '')
+
+{%- elif target.type == 'postgres' -%}
+replace(json_extract_path_text(event_properties, {{ "'" ~ property ~ "'" }} ), '"', '')
```
Comment (fivetran-joemarkiewicz) [inline macros/pivot_event_properties_json.sql:14]: Can we do either of the below instead?
- Combine the redshift and postgres conditional to be `{%- elif target.type in ('redshift', 'postgres') -%}
- Is this a macro that our [current json_extract](https://github.com/fivetran/dbt_fivetran_utils/blob/master/macros/json_extract.sql) could handle? Or should we consider moving this macro to our dbt_fivetran_utils?

**Review by fivetran-joemarkiewicz — COMMENTED** [review]
Looks good! I just had a few comments before merging

File: macros/pivot_event_properties_json.sql (hunk 2 above)
Comment (fivetran-jamie) [reply macros/pivot_event_properties_json.sql:14]: ah that's a good call out -- i've always felt like this belongs in fivetran_utils so i'll clean it up + move it there

File: packages.yml [hunk 3]
```diff
@@ -4,3 +4,4 @@ packages:
   
   - git: "https://github.com/fivetran/dbt_fivetran_utils.git"
     warn-unpinned: false
+    revision: jr_working
```
Comment (fivetran-jamie) [inline packages.yml:7]: will update after this is closed https://github.com/fivetran/dbt_fivetran_utils/pull/24

**Pull request merged**